icu4j/main/core/src/test/java/com/ibm/icu/dev/test/lang/UnicodeSetTest.java

```diff
@@ -61,6 +61,13 @@ public class UnicodeSetTest extends CoreTestFmwk {
 
     static final String NOT = "%%%%";
 
+    // Used to test iterators and streams
+    final static UnicodeSet[] UNICODE_SETS_TO_ITERATE = {
+            UnicodeSet.EMPTY,
+            // UnicodeSet.ALL_CODE_POINTS, // this works, but the tests are visibly slow
```

Comment:
faster without charAt()?
if you don't intend to replace this with another set, then i suggest deleting this comment -- it's not unusual that doing something for all 1.1M Unicode code points would be slow

Reply:
Done

It is faster without `charAt`, it is even a bit faster than the older `UnicodeSetIterator`

Interestingly enough, I don't see a significant speed difference even with `Iterator<Integer>` (boxed values instead of primitives).
I know that Java improved boxing performance bit time, but I did not expect it to be un-noticeable.